src/components/Notification/icons/index.js

```diff
@@ -23,7 +30,7 @@ export default function Icon({ icon }) {
     if (typeof icon === 'string') {
         return (
             <div className={getClassName()}>
```

Comment:
sorry, it is better if we return this container inside VariantIcon, thus if a wrong string is passed it is not rendered. So VariantIcon would be:
```
function VariantIcon({ icon }) {
    if (iconMap[icon]) {
        return (
            <div className={getClassName()}>
                {iconMap[icon]()}
            </div>
        );
    }
    return null;
}
```
Also you need to test this case (when a wrong string is passed), that null is returned or nothing is rendered.